database/redis/README.md

```diff
@@ -0,0 +1,149 @@
+
+In this example, we will generate a Singularity container running Redis server v5.0.0 over Debian 9 (Stretch).
+
+To run this example you will need:
+
+ - Singularity, which you can download and install from [here](https://github.com/sylabs/singularity).
+ - A text editor, like: `micro`, `vim`, or `nano`.
+ - Root access.
+
+We'll go through the explanation of every section in the definition file:
+
+In the `%help` section, you can find the description for the container with the respective version of Redis installed (5.0.0) for the specific OS distribution.
+
+In the `%post` section, all the needed dependencies are installed to make Redis work properly.
+
+In the `%startscript` section, the commands needed to run an instance of the Redis server are called. We will use this section to start the Redis server daemon by default in `127.0.0.1:6379`.
+
+In the `%runscript` section, the commands needed to run a container are called. In this case we will use this section to call the Redis Client and interact with the database.
+
+#### Build your Redis container using Singularity:
+
+We will work in a directory called ` ~/redis`, if you don't have it create it and inside get the definition file:
+
+```
+$ mkdir  ~/redis
+$ cd  ~/redis
+```
+
+To obtain the definition file just run:
+
+```
+$ wget https://raw.githubusercontent.com/sylabs/examples/master/database/redis/redis.def
+```
+
+After that you could simply run the following command:
+
+```
+$ sudo singularity build redis.sif redis.def
+```
+
+#### Build your Redis container using the Singularity Remote Builder:
+
+If you do not own root permissions you can always build your Redis Singularity container with the help of our [remote builder](https://cloud.sylabs.io/builder) like so:
+
+```
+$ singularity build --remote redis.sif redis.def
+```
+
+If this is the first time using the Remote Builder, use the following
+[instructions](https://cloud.sylabs.io/auth) to create an identification token. Your container will be built on the Remote Builder and all standard out
+and standard err will be directed back to your terminal. When finished, the
+container will be automatically downloaded to your local machine.
+
+If you do not have Singularity installed on your machine, or you are in a non-Linux environment, you can also make use of the remote builder. For this, you should sign into the Sylabs Cloud and compose your definition files directly in the main window from  [Remote Builder](https://cloud.sylabs.io/builder) page. You can also drag and drop text files there. The container you build will appear under your username in the `remote-builds` collection. After this, you can download your container with the `pull` command explained below.
+
+You can also pull the container from our library like so:
+
+```
+$ singularity pull library://sylabs/examples/redis:latest
+```
```

Comment:
Maybe add this after the pull command:

<br>

then change the container name:
```
$ mv redis_latest.sif redis.sif
```

<br>

Because the pulled container will have `_latest.sif` at the end.